### DISCUSSION TITLE
Sidecar injection is not working

### DISCUSSION THREAD

**Discussion by K-R-Swarna** [istio/istio#50953] [Q&A]
Prerequisites: Istio-base, istiod, istio-ingress-gateway helm chart is being deployed.

I tried implementing the below scenarios for the deployment.yaml, but pods are still running with 1/1 which means service mesh is not getting enabled,

1. labels: sidecar.istio.io/inject: "true" 
2. annotations:
        sidecar.istio.io/inject: "true"
        sidecar.istio.io/rewriteAppHTTPProbers: "false"
3. annotations:
        sidecar.istio.io/inject: "false"

**Comment by craigbox** [comment]
Can you share your YAML?  Is there a reason you are not [enabling injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/) on an entire namespace?
